src/cmd/singularity/cli/capability_add.go

```diff
@@ -30,14 +26,10 @@ func init() {
 	// -d|--desc
 	capabilityAddFlags.BoolVarP(&CapDesc, "desc", "d", false, "Print capabilities description")
 
-	for _, cmd := range capabilityAddCmds {
-		cmd.Flags().AddFlag(capabilityAddFlags.Lookup("user"))
-		cmd.Flags().AddFlag(capabilityAddFlags.Lookup("group"))
-		cmd.Flags().AddFlag(capabilityAddFlags.Lookup("desc"))
-		cmd.Flags().SetInterspersed(false)
-	}
-
-	// SingularityCmd.AddCommand(capabilityDotAddCmd)
+	CapabilityAddCmd.Flags().AddFlag(capabilityAddFlags.Lookup("user"))
```

Comment:
I think for the capability commands it's okay to directly add the flag to the `Cmd.Flags()` directly. We have the `actionFlags` variable because multiple commands needed to share the same set of flags.

So I would say we can simply do:

`CapabilityAddCmd.Flags().StringVarP(...)` etc...

Same for the other commands that don't share flags with other commands.

Reply:
I think these are shared between subcommands too.  So this avoids a bit of redundancy.  Right?

Reply:
specifically the `add`, `drop`, and `list` subcommands afaik

Reply:
Nevermind.  I'm dumb.  🤤